src/components/ProjectInformation/index.tsx

```diff
@@ -0,0 +1,75 @@
+import githubLogo from "assets/img/github-logo.svg";
+import telegramLogo from "assets/img/telegram-logo.svg";
+import onlyDustLogo from "assets/img/onlydust-logo.png";
+
+export const REMAINING_AMOUNT_KEY = "remaining_amount";
+export const INITIAL_AMOUNT_KEY = "initial_amount";
+export const TELEGRAM_LINK_KEY = "telegram_link";
+
+interface ProjectInformationProps {
+  name: string;
+  details?: {
+    description: string;
+    [TELEGRAM_LINK_KEY]: string;
```

Comment:
I think we shouldn't receive data with snake case formatting, it should be dealt with in hasura or when receiving data (like this for example : https://stackoverflow.com/a/32441713/10261061)